plugins/woocommerce/tests/php/includes/tracks/class-wc-tracks-client-test.php

```
@@ -0,0 +1,420 @@
+		// Ensure batching is supported.
+		if ( ! class_exists( 'WpOrg\Requests\Requests' ) && ! class_exists( 'Requests' ) ) {
+			$this->markTestSkipped( 'Requests library not available for batching.' );
+		}
+
+		$pixel = 'https://pixel.wp.com/t.gif?_en=test_event';
+
+		// Queue a pixel.
+		WC_Tracks_Client::record_pixel( $pixel );
+
+		// Mock the send_with_requests_multiple method to capture the pixels.
+		$sent_pixels = array();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sent_pixels'. (undefined)

(UnusedLocalVariable)